ggml/src/ggml-rpc.cpp

```diff
@@ -96,6 +95,77 @@ enum rpc_cmd {
     RPC_CMD_COUNT,
 };
 
+#pragma pack(1)
+struct rpc_msg_alloc_buffer_req {
+    uint64_t size;
+};
```

Comment:
this is not really how `pack(1)` works. every struct after a single `pack(1)` will be affected, only after another `pack()` (empty), it will be the value from before.

Reply:
Nice catch, I have posted #9959 to fix this